src/main/java/io/github/jhipster/registry/web/rest/JWTToken.java

```diff
@@ -1,24 +0,0 @@
-package io.github.jhipster.registry.web.rest;
-
```

Comment:
Ok @deepu105, I'll make a PR for the generator soon.
So, you want (on the Registry) that I revert this commit & remake a PR ?